lib/zlib.js

```diff
@@ -326,15 +332,8 @@ function Zlib(opts, mode) {
     }
   }
 
-  if (opts.strategy) {
-    if (opts.strategy != constants.Z_FILTERED &&
-        opts.strategy != constants.Z_HUFFMAN_ONLY &&
-        opts.strategy != constants.Z_RLE &&
-        opts.strategy != constants.Z_FIXED &&
-        opts.strategy != constants.Z_DEFAULT_STRATEGY) {
-      throw new Error('Invalid strategy: ' + opts.strategy);
-    }
-  }
+  if (opts.strategy && !(strategies.includes(opts.strategy)))
```

Comment:
Nit: why `!(strategies.includes(opts.strategy))` instead of just `!strategies.includes(opts.strategy)`